routers/repo/issue.go

```diff
@@ -453,6 +454,8 @@ func NewIssuePost(ctx *context.Context, form auth.CreateIssueForm) {
 		return
 	}
 
+	notification.Service.NotifyIssue(issue)
```

Comment:
You have a good point. Channels act like a queue, so I think it will block until it's consumed by the service.

We can use buffered channels [1], but I think we will have a panic if we exceed the buffer size. I will make some tests.

[1] https://gobyexample.com/channel-buffering